test/tokentree/utils/FieldUtilsTest.hx

```diff
@@ -57,7 +57,7 @@ class FieldUtilsTest {
 
 @:enum
 abstract FieldUtilsTests(String) to String {
-	var PROPERTIES = "
+	var PropERTIES = "
```

Comment:
Hmm... :)

Reply:
argl... search and replace gone wrong.... 
we should really look into refactoring for vshaxe ;)